k8s/custom-governance/cli_values_template.yaml

```diff
@@ -1,12 +1,12 @@
 # # This is a YAML-formatted file.
 # # Declare variables to be passed into your templates.
-aditum:
+cg:
   image:
     # Container registry repo where the application image is hosted.
     # Typically you do not need to modify this.
-    repo: marketplace.gcr.io/aditum-marketplace-dev/custom-governance
+    repo: gcr.io/cloud-marketplace/custom-governance/custom-governance
```

Comment:
This should be equivalent to `marketplace.gcr.io/custom-governance/custom-governance`. I would recommend using the `marketplace.gcr.io` if you can confirm that it works.

The problem before appears to be the incorrect/old `aditum-marketplace-dev` in the path.